Creating tribute

sodabarauf opened the pull request:
this project is about making a tribute page, in this project I did these things:
#first of all, I followed the GitHub workflow which include creating repository, creating development branch and feature branch, clone repository with local machine, work on feature branch, set up linters config, work and commit, push commits to GitHub....
#i added two new files index.html and style.css.
#i linked HTML with CSS file
#i added main tag, h1 tag, h3 tag, and h5 tag to the index.html file.
#i added div tags, img tag, fig caption tag, ul and li tags.
#i added some styles in the CSS file.
#i added the linters config of html and css.

ahzamir commented on index.html:
```diff
@@ -0,0 +1,58 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>tribute page</title>
+    <link rel="stylesheet" href="style.css">
+    <style>
+        #image {
+          max-width: 100%;
+          height: auto;
+        }
+
+        img {
+            display: block;
+        }
+        
+    </style>
```
- You have a separate file for adding your style codes so remove and separate the style codes from your HTML file.

ahzamir commented on index.html:
```diff
@@ -0,0 +1,58 @@
+    </style>
+</head>
+<body>
+    
+    <main id="main">
+        <h1 id="title">Rumi(Maulana Jalal ad-Din Muhammad)</h1>
+        <p>  "Beauty surrounds us, but usually we need to be walking in a garden to know it."</p>
+    
+        <div id="img-div">
+            <img id="image" alt="Rumi image" src="https://assets.ltkcontent.com/images/184325/Rumi-Quotes_b455f1d160.jpg" width="100%" height="auto" />
+            <figcaption id="img-caption">Jalal ad-Din Muhammad commonly known as Rumi, A persian poet, mystic, islamic scholar and sufi philosopher who was born in Balkh(Afghanistan). Rumi's work have had a profound impact on literature, philosophy, and spirituality, His poetry is widely celebrated for its spiritual wisdom and universal themes.</figcaption>
+            </div>
+    
+            <div id="tribute-info">
+                <h3>Here is time line of Rumi: </h3>
+                <ul>
+                    <li><strog>Meaning of Rumi: </strog>This is a name which comes from arabic and literally means"Roman" because he spent much of his life in the Anatolia which was area of Eastern Roman.</li>
+                    <li><strog>Early Life: </strog>He was born in 1207 in Balkh in Afghanistan, Jalaluddin's father, Baha al-Din Walad was a noted mystical theologian, author, and teacher. because of either a dispute with roles of approaching Mongols, his family left their native town and started journeys, till reached in Konya and resided there.</li>
+                    <li><strong>Influence of Shams ad-Din: </strong>The decisive moment in Rūmī’s life occurred on November 30, 1244, when in the streets of Konya he met the wandering dervish—holy man—Shams al-Dīn (Sun of Religion) of Tabrīz, whom he may have first encountered in Syria. Shams al-Dīn cannot be connected with any of the traditional mystical fraternities; his overwhelming personalit ...
+                    <li><strong>Works: </strong>Rumi is best known fot his poetry collections, particularly the "Masnavi", and the "Divan-e-shams-e Tabrizi".</li>
+                    <li><strong>Masnavi Ma'anavi: </strong>The Masnavi is a poetic collection of rambling anecdotes and stories derived from the Quran, Hadith sources, and everyday tales. The Masnavi is a series of six books of poetry that amount to about aprox. 27,000 verses or 54,000 lines. The title Masnavi-Ma'navi means "Rhyming Couplets of Profound Spiritual Meaning." it is a spiritual writi ...
+                    <li><strong>Quotes: </strong>
+                      <ol>
+                          <li>“Be with those who help your being.” ...</li>
+                          <li>“It is your light that lights the world.” ...</li>
+                          <li>“Let the beauty of what you love be what you do.” ...</li>
+                          <li>“The garden of the world has no limits, except in your mind.” ...</li>
+                          <li>“Everything in the universe is within you. Ask all from yourself.”</li>
+                      </ol>
+                    </li>
+                    <li><strong>Death and Legacy: </strong>Rūmī lived for a short while after completing the Mas̄navī.and died on 17 December 1273. He always remained a respected member of Konya society,His burial procession, according to one of Rūmī’s contemporaries, was attended by a vast crowd of people of many faiths and nationalities. His mausoleum, the Green Dome, is today a museum in Konya ...
+                    </ul>
+                    </div>
+                
+                <h5>
+                        If you have time, you should read more about this incredible person being on his<a id="tribute-link" target="_blank" href="https://www.WIKIpedia.com" rel="noopener">WIKIpedia</a>
+                </h5>
+          </main>
+    
+</body>
```
- Your body spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

ahzamir reviewed (changes requested):
### Hi @sodabarauf 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although your code passes all checks on freecodecamp website on the tribute page project, but I added some inline comments **(Check below)** for you to implement for clean code.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

![AlmostThereReachGIF](https://github.com/LimaSahibzada/Survey-Form-project1/assets/96838030/4f0a3cd6-3a46-4e1e-8e31-ef617d4e8ac3)

 ## Highlights 🌞
 
🎯 Great Design implemented.
![image](https://github.com/sodabarauf/tribute/assets/96838030/cef28590-5cf6-44ec-b019-c3ed1ae33de6)
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Tribute Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your code is clean.

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

sodabarauf commented:
@ahzamir, sir thanks from your feedback, beside external CSS I added internal CSS because in freecodecamp.org, there showed error, so I added, now removed the internal CSS and hope be acceptable.
also, I cleaned the code according to the requirement which you have provided to me.
thank you very much sir.

Commit:
added index.html and style.css files

Commit:
added inters to the project

Commit:
fixed the linters issue

ahzamir commented:
@sodabarauf 
- Still, your code is not clean in the HTML file also delete the `HEAD>>>>>>>` and other unnecessary lines from your code
- And for cleaning you code:
  - At the bottom right side of your vscode click on `Spaces` then click on `Index using spaces` and choose 2 from there.
  - Then according to the lines on the screen all child elements should be 2 spaces or one line ahead of their parent element and the closing tag should be in the same line as the opening tag!

Commit:
added new <li> tag

Commit:
added main tag into body tag also h1 tag and p tag into main tag

Commit:
added img tag and figcaption tag into div tag in the main tag

Commit:
added new div tag, and added h3 tag, unorder list tag with <li> tags in the new div tag

Commit:
added h5 tag and some semantig tag like <strong> into main tag

Commit:
added some ptoperties in style.css file to index.html file

sodabarauf commented:
@ahzamir 
sir, thank you. 
now, please review this.

Commit:
cleaned the code

Commit:
cleaned the code

Commit:
cleaned the code

Commit:
cleaned the code

Commit:
cleaned the code

sodabarauf commented:
@ahzamir sir please review my changes which i worked again and provide feedback if possible,
Because by adding these changes I think it has caused some problem.

ahzamir reviewed (approved):
### Hi @sodabarauf 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
![Screenshot 2024-05-22 114625](https://github.com/sodabarauf/tribute/assets/96838030/7b9d15e3-a4f4-4c27-9e0f-151bdaaa9850)



# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

![GreatJobThumbsUpGIF](https://github.com/ZuhRa-HashiMi/survey-form/assets/96838030/cc459d3f-5796-43f0-aba7-4ed0c35fa276)

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Tribute Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

sodabarauf commented:
@ahzamir sir, I appreciate from your struggles in this part with us.
thank you.
